iocore/net/QUICNetVConnection.cc

```diff
@@ -1370,6 +1369,8 @@ QUICNetVConnection::_packetize_frames(QUICEncryptionLevel level, uint64_t max_pa
   }
 
   bool ack_only = true;
+  uint8_t frame_instance_buffer[1024]; // This is for a frame instance but not serialized frame data
```

Comment:
I hope 1024 is enough, but nobody knows how much we need. A new frame class might need more. Thoughts?

Reply:
Why we can't just declare `QUICFrame` on stack?

Reply:
Never mind, I forgot `QUICFrame` is abstraction class.